client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -159,7 +159,7 @@ const ballotMetadata = ({
     ['bs', ballotStyleId],
     ['p', `${pageNumber}-${pageCount}`],
   ])
-  return new URL(`https://vx.vote/?${params}`).toString()
+  return new URL(`https://track.vote/?${params}`).toString()
```

Comment:
I don't think we want something in the *ballot* that implies tracking. I know we wouldn't be, but someone reading that QR code could get really scared. We should think about what this URL should be, but probably not `track.vote`.

Reply:
Ok, what should url be? We don't (yet!) own `vx.vote`.

Reply:
`ballot.page`